protocol/communities/manager.go

```diff
@@ -1032,12 +1045,21 @@ func (m *Manager) ImportCommunity(key *ecdsa.PrivateKey) (*Community, error) {
 
 	if community == nil {
 		description := &protobuf.CommunityDescription{
-			Permissions: &protobuf.CommunityPermissions{},
+			Permissions: &protobuf.CommunityPermissions{
+				Access: protobuf.CommunityPermissions_ON_REQUEST,
```

Comment:
Is this a leftover of testing?

Reply:
No, it's intentional. Without it, [the validation](https://github.com/status-im/status-go/blob/17cef5f211b6bd599592cbc27bcf40907688d238/protocol/communities/community.go#L943) for 'CommunityDescription' will fail, leading to all community updates being rejected by clients. I discovered this when my newly-written test failed due to an invalid 'CommunityDescription'

P.S. I bet imported communities never truly functioned 🤷‍♂️

Reply:
That's weird. We really should test imported communities. This looks like it makes it so that all imported communities become on request. It's not the biggest deal, but still.

Reply:
Among all options, `CommunityPermissions_ON_REQUEST` seems as a good default when importing without previous community information. I am open to change it to something else if that makes more sense.